moveit_ros/benchmarks/CHANGELOG.rst

```diff
@@ -1,121 +0,0 @@
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
We have replaced the old ``benchmarks`` package with an entirely new one (code/method wise). I'm not sure if we would want to carry over the old CHANGELOG to the new package. I would vote no since the old CHANGELOG wouldn't have anything relevant to say about the new package and would probably just confuse users if it promises some new feature, etc